modules/installation-uninstall-clouds.adoc

```diff
@@ -126,11 +142,18 @@ $ ccoctl ibmcloud delete-service-id \
 If your cluster uses Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set, you must include the `--enable-tech-preview` parameter.
 ====
 --
-endif::ibm-cloud[]
+endif::ibm-cloud,ibm-power-vs[]
 
 . Optional: Delete the `<installation_directory>` directory and the
 {product-title} installation program.
 
+ifdef::ibm-power-vs[]
+[NOTE]
+====
+You might have to run the openshift-install destroy command up to three times to ensure a proper cleanup.
```

Comment:
s/openshift-install destroy/`openshift-install destroy`

Reply:
Should this note be under Step 3, where you use the command?

Reply:
Yes, I agree. Have updated the module accordingly. Thanks @mburke5678

Reply:
@alishaIBM Please add backticks to the command in this note 

`You might have to run the `openshift-install destroy` command...`